railties/lib/rails/generators/app_base.rb

```diff
@@ -422,7 +422,7 @@ def run_bundle
       end
 
       def run_webpack
-        if !(webpack = options[:webpack]).nil?
+        if bundle_install? && !(webpack = options[:webpack]).nil?
```

Comment:
Almost user whose create app with `--skip-bundle` and `--webpack` options may want to execute install manually via `bin/rails webpacker:install`. So it's not happy to raise error. And I did implementation which ignores as same as spring binstub.